src/gateways/googleAnalyticsClient.ts

```diff
@@ -196,18 +213,20 @@ class GoogleAnalyticsClient {
       return;
     }
 
-    for (const tagId of this.tagIds) {
+    for (const tag of this.tags) {
       command.payload = {...command.payload, queue_time: queueTime };
+
       if (command.name === 'event') {
-        this.gtag('event', command.eventName, { ...command.payload, send_to: tagId });
+        if (tag.ignoredEventNames.includes(command.eventName)) { continue; }
```

Comment:
you could invert the condition and put the gtag call inside it, that feels a bit nicer than using the continue to short circuit